.github/workflows/release.yml

```diff
@@ -41,6 +41,8 @@ jobs:
     - name: Create github release
       uses: softprops/action-gh-release@de2c0eb89ae2a093876385947365aca7b0e5f844 # tag v0.1.15
       if: $(git tag -l ${{ env.VERSION }}) == false
+      permissions:
+        contents: write
```

Comment:
This actions requires `content: write` permissions
https://github.com/softprops/action-gh-release#permissions